modules/gapi/samples/pipeline_modeling_tool.cpp

```diff
@@ -326,7 +335,7 @@ int main(int argc, char* argv[]) {
                     }
                     auto output =
                         check_and_read<OutputDescr>(node_fn, "output", node_name);
-                    builder.addDummy(node_name, time, output);
+                    builder.addDummy(node_name, call_every_nth_u, time, output);
```

Comment:
how's about making struct for `call_every_nth_u` and further possible params?

Reply:
Do you mean like this:
```
builder.addDummy(NodeParams{node_name, call_every_nth_u}, time, output);
```

Reply:
👍🏻  yes, something like that

Reply:
Done